features/api_security_status_package_manifest.feature

```diff
@@ -0,0 +1,40 @@
+Feature: API security status package list in manifest format
+
+    @series.lts
+    @uses.config.machine_type.aws.pro
+    @uses.config.machine_type.azure.pro
+    @uses.config.machine_type.gcp.pro
```

Comment:
I don't think this test needs to run on pro cloud instances, probably just lxd containers.
But otherwise this is a great test!